### PR TITLE
[en] sync cluster-ns-auth

### PR DESCRIPTION
**Pull request by ModetaNiu**
补充英文版

### CODE REVIEW

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Increase Kpanda Built-in-role Rbac Rules
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:1]:
```suggestion
# Add RBAC Rules in Container Management
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+*[kpanda]: A development codename for container management
+   
+The permission points (rbac rules) for kpanda's built-in roles are pre-defined and cannot be modified by users. Any changes made to the built-in role permissions will be reverted to the predefined permissions by the kpanda controller. To support more flexible permission configurations and to meet the customized needs for system roles, kpanda allows the modification of permission points for built-in system roles (cluster admin, ns admin, ns edit, ns view).
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:5]:
```suggestion
The RBAC rules for those built-in roles in container management are pre-defined and cannot be modified by users.
Previously, any changes made to the rules will be reverted.
To support more flexible permission settings and to meet the customized needs for system roles,
now you can modify allows  RBAC rules for system roles such as cluster admin, ns admin, ns editor, ns viewer.
```

**Review by windsonsea — COMMENTED** [review]
- Need many more efforts to polish those text.
- Provide newline around 100-120 col for the purpose of review convenience.

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+# Adding RBAC Rules in Container Management
+
+*[kpanda]: A development codename for container management
+   
+The RBAC rules for those built-in roles in container management are pre-defined and cannot be modified by users.
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:5]:
```suggestion
In the past, the RBAC rules for those built-in roles in container management were pre-defined and could not be modified by users.
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
+# Adding RBAC Rules in Container Management
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:1]:
```suggestion
# Adding RBAC Rules to System Roles
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        ```output
+        cluster-admin: rbac.kpanda.io/role-template-cluster-admin: "true"
+        cluster-edit: rbac.kpanda.io/role-template-cluster-edit: "true"
+        cluster-view: rbac.kpanda.io/role-template-cluster-view: "true"
+        ns-admin: rbac.kpanda.io/role-template-ns-admin: "true"
+        ns-edit: rbac.kpanda.io/role-template-ns-edit: "true"
+        ns-view: rbac.kpanda.io/role-template-ns-view: "true"
+        ```
+
+## Operation Steps
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:34]:
```suggestion
## Steps
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+  [created Kubernetes cluster](../clusters/create-cluster.md), and able to access the cluster's UI interface.
+- Completed creation of a [namespace](../namespaces/createns.md) and [user account](../../../ghippo/user-guide/access-control/user.md), 
+  and the granting of [NS Viewer](./permission-brief.md#ns-viewer).
+  For details, refer to [namespace authorization](./cluster-ns-auth.md).
+
+
+!!! note
+
+    - RBAC rules **only need to be added** in the Global Cluster, and the Kpanda controller will synchronize 
+      those added rules to all connected subclusters. Synchronization may take some time to complete.
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:28]:
```suggestion
      those added rules to all integrated subclusters. Synchronization may take some time to complete.
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+        cluster-edit: rbac.kpanda.io/role-template-cluster-edit: "true"
+        cluster-view: rbac.kpanda.io/role-template-cluster-view: "true"
+        ns-admin: rbac.kpanda.io/role-template-ns-admin: "true"
+        ns-edit: rbac.kpanda.io/role-template-ns-edit: "true"
+        ns-view: rbac.kpanda.io/role-template-ns-view: "true"
+        ```
+
+## Steps
+
+1. Users with admin or cluster admin permissions [create stateless workloads](../workloads/create-deployment.md)
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:46]:
```suggestion
1. [Create a deployment](../workloads/create-deployment.md) by a user with `admin` or `cluster admin` permissions.
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+    ![image-20240514112742395](../images/create-depolyment.png)
+
+1. Users grant ns-viewer and posess namespace ns-view authorization
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:50]:
```suggestion
1. Grant a user the `ns-viewer` role to provide them with the `ns-view` permission.
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+  
+    ![image-20240514113009311](../images/permisson02.png)
+
+1. Switch the login user to ns-viewer, open the control panel to get the token corresponding to the ns-viewer user,
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:54]:
```suggestion
1. Switch the login user to ns-viewer, open the console to get the token for the ns-viewer user, 
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+1. Users with admin or cluster admin permissions [create stateless workloads](../workloads/create-deployment.md)
+
+    ![image-20240514112742395](../images/create-depolyment.png)
+
+1. Users grant ns-viewer and posess namespace ns-view authorization
+  
+    ![image-20240514113009311](../images/permisson02.png)
+
+1. Switch the login user to ns-viewer, open the control panel to get the token corresponding to the ns-viewer user, 
+   and use curl to request to delete the above deployment nginx, finding that there is no deletion permission.
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:55]:
```suggestion
   and use `curl` to request and delete the nginx deployment mentioned above. However,
   a prompt appears as below, indicating the user doesn't have permission to delete it.
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+    {"code":7,"message":"[RBAC] delete resources(deployments: nginx) is forbidden for user(ns-viewer) in cluster(cluster-member)","details":[]}[root@master-01 ~]#
+    [root@master-01 ~]#
+    ```
+
+1. Create the following ClusterRole on the global cluster, as shown in the yaml below.
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:63]:
```suggestion
1. Create a ClusterRole on the global cluster, as shown in the yaml below.
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+      - apiGroups: [ "apps" ]
+        resources: [ "deployments" ]
+        verbs: [ "delete" ]
+    ```
+
+    1. This field value can be arbitrarily specified, as long as it is not duplicated and complies with 
+       the Kubernetes resource naming conventions.
+    2. When adding rules to different roles, make sure to apply different labels.
+
+1. Wait for the kpanda controller to add the user creation rule to the built-in role ns-view. Check
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:82]:
```suggestion
1. Wait for the kpanda controller to add a rule of user creation to the built-in role: ns-viewer. Check 
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+        resources: [ "deployments" ]
+        verbs: [ "delete" ]
+    ```
+
+    1. This field value can be arbitrarily specified, as long as it is not duplicated and complies with 
+       the Kubernetes resource naming conventions.
+    2. When adding rules to different roles, make sure to apply different labels.
+
+1. Wait for the kpanda controller to add the user creation rule to the built-in role ns-view. Check 
+   the proper built-in role to see if the rules added in the previous step are present.
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:83]:
```suggestion
   then you can check if the rules added in the previous step are present for ns-viewer. 
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+    ```yaml
+    - apiGroups:
+      - apps
+      resources:
+      - deployments
+      verbs:
+      - delete
+    ```
+
+1. When using curl again to request the deletion of the aforementioned deployment nginx, this time the deletion
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:97]:
```suggestion
1. When using curl again to request the deletion of the aforementioned nginx deployment, this time the deletion 
```

File: docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+    - apiGroups:
+      - apps
+      resources:
+      - deployments
+      verbs:
+      - delete
+    ```
+
+1. When using curl again to request the deletion of the aforementioned deployment nginx, this time the deletion 
+   was successful. This means that ns-viewer has successfully added the rule to delete Deployments.
```
Comment (windsonsea) [inline docs/en/docs/kpanda/user-guide/permissions/custom-kpanda-role.md:98]:
```suggestion
   was successful. This means that ns-viewer has successfully added the rule to delete deployments.
```

**Review by windsonsea — COMMENTED** [review]
`system role` is same as `built-in role`. We can pick one and use it consistently.

**Comment by ModetaNiu** [comment]
> `system role` is same as `built-in role`. We can pick one and use it consistently.

Ok I see.

**Pull request merged**